### DISCUSSION TITLE
cx-flow with async scan

### DISCUSSION THREAD

**Discussion by dszortyka** [checkmarx-ltd/cx-flow#1149] [Q&A] [answered]
Hello,

We are using cx-flow to trigger manual scans on checkmarx. I was looking the new "async" feature mentioned in the recent release. 

> "Ability to run async scan that returns scan-id and then run the cxflow command later in pipeline and fetch/wait for results based on scanId. @itsKedar"

Does anyone know how to make this call using async option? 

Thanks! 
-Daniel

**Comment by marciacx** [comment, marked as answer]
Hello Daniel, 
To provide async behavior to cx-Flow, bug-tracker should be set as **NONE**. Check here for more details: [https://github.com/checkmarx-ltd/cx-flow/wiki/Bug-Trackers-and-Feedback-Channels#none](url)

We also provide scan-id of scan in logs.

Now if there is a need to process the result for same scan we have added a new CLI parameter in project mode –scanId which is  mentioned [here](https://github.com/checkmarx-ltd/cx-flow/wiki/Execution).

Let us know if you need further clarifications and we will be happy to help.
